app/models/campaign.rb

```diff
@@ -104,6 +118,19 @@ def seller_distributor_pairs
 
   private
 
+  def mega_gam?
+    project.present?
+  end
+
+  # Calculates the amount raised in the payment intent table.
+  def payment_intent_amount
+    PaymentIntent
+      .joins(:campaign)
+      .where('campaigns.id=? AND successful=true', id)
+      .map { |payment_intent| payment_intent.amount }
```

Comment:
nit but you can do:
`.map { &.amount }`

Reply:
that syntax isn’t working for me:
```SyntaxError:
  /Users/chrisng/dev/github/ruby/app/models/campaign.rb:133: syntax error, unexpected &.
        .map { &.amount }```